fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreIndexTest.java

```diff
@@ -2286,30 +2287,38 @@ public void formerIndexes() throws Exception {
 
     @Test
     public void boundaryPrimaryKeys() throws Exception {
```

Comment:
This PR should probably revert the change from `assertTrue` -> `assumeTrue` in #567.

Reply:
Good point. Thanks for pointing out.